docs/cookbook/building-navigation.md

```diff
@@ -0,0 +1,318 @@
+# Building site navigation
+
+Almost every website needs some kind of primary navigation. It's usually a row of links at the top of each page that is hidden behind some kind of "hamburger menu" button on smaller devices. For Apostrophe sites there are two main approaches to adding and maintaining site navigation:
```

Comment:
I think you can remove this sentence: "It's usually a row of links at the top of each page that is hidden behind some kind of "hamburger menu" button on smaller devices."